spec/features/shelters/reviews/new_spec.rb

```diff
@@ -0,0 +1,91 @@
+RSpec.describe "As a visitor:" do
```

Comment:
changing the URI made moving this test to this directory make more sense. the shelter show still checks to see it displays a list of reviews, all other review functionality testing goes in /spec/features/shelters/reviews